client/dashboard/profile-wizard/style.scss

```diff
@@ -140,3 +141,26 @@
 		}
 	}
 }
+
+.woocommerce-profile-wizard__header.is-stepper {
+	display: block;
```

Comment:
This removes the vertical centering for the stepper.  I think the intention behind this was to get the stepper to full width to show the divider lines in which case we could use this as a substitute:

```scss
.woocommerce-stepper {
    width: 100%;
}
```